eth/stagedsync/stages/stages.go

```diff
@@ -89,6 +89,10 @@ func GetStageProgress(db kv.Getter, stage SyncStage) (uint64, error) {
 }
 
 func SaveStageProgress(db kv.Putter, stage SyncStage, progress uint64) error {
+	fmt.Println("SaveStageProgress", stage, progress)
```

Comment:
Would be better to use logger here so that it shows up on the `.log` files

Reply:
I forgot to remove this log, thanks